src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -1111,6 +1106,37 @@ private ConnectorParameterInternals Initialize()
             };
         }
 
+        private class OpenApiParameterComparer : IEqualityComparer<OpenApiParameter>
+        {
+            public bool Equals(OpenApiParameter x, OpenApiParameter y)
+            {
+                if (x == null && y == null)
+                {
+                    return true;
+                }
+
+                if (x == null || y == null)
+                {
+                    return false;
+                }
+
+                return (x.Schema == y.Schema) && 
+                       (x.Name == y.Name) &&
+                       (x.Required == y.Required) &&
+                       (x.Extensions == y.Extensions);                
```

Comment:
> (x.Extensions == y.Extensions);        [](http://example.com/codeflow?start=22&length=39)

Extensions is a IDictionary<string, IOpenApiExtension>. 

What does this mean?